src/Symfony/Component/Mercure/Publisher.php

```diff
@@ -80,16 +87,32 @@ private function encode($key, $value): string
 
     private function publish(string $url, string $jwt, string $postData): string
     {
-        $result = @file_get_contents($this->publishEndpoint, false, stream_context_create(array('http' => array(
+        $result = @file_get_contents($this->hubUrl, false, stream_context_create(array('http' => array(
             'method' => 'POST',
             'header' => "Content-type: application/x-www-form-urlencoded\r\nAuthorization: Bearer $jwt",
             'content' => $postData,
         ))));
 
         if (false === $result) {
-            throw new \RuntimeException(sprintf('Unable to publish the update to the Mercure hub: %s', error_get_last()));
+            throw new \RuntimeException(sprintf('Unable to publish the update to the Mercure hub: %s', error_get_last()['message'] ?? 'unknown error'));
         }
 
         return $result;
     }
+
+    /**
+     * Regex ported from Windows Azure Active Directory IdentityModel Extensions for .Net.
```

Comment:
s/.net/.NET/

Reply:
I copied the name of the project from their README: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet#windows-azure-active-directory-identitymodel-extensions-for-net